### PR TITLE
1.0.0 doc changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
- Rewrote usage, readme, and setup
- Added navigation links

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -3,61 +3,78 @@
...
+            <li><code>data</code></li>
+            <li><code>title</code> and <code>value</code></li>
+        </ul>
+    </td>
+</table>
+
+## Record Summary Tile
+
+1. Edit a *Dashboard*, *Report*, or a module record's view template and select **Add Widget** button.
+
```
Comment (cs-yash-patodkar) [inline docs/usage.md:50]: This widget Does not work for Reports or module records view template, Just keep dashboard

File: README.md [hunk 2]
```diff
@@ -12,10 +12,14 @@
 
 
 ## Overview
-To use this widget users needs to provide a JSON in a certain format for the widget to render data.
-This JSON data must be in a field of a record in a module.
 
+**Introducing Record Summary Tile: Your FortiSOAR&trade; MSSP Data at a Glance**
 
+Experience the power of simplified data aggregation and visualization with the **Record Summary** widget, exclusively designed for FortiSOAR&trade;. This innovative widget transforms complex JSON data into concise, informative tiles, providing SOC analysts and managers a clear snapshot of critical information.
+
```
Comment (cs-yash-patodkar) [inline README.md:19]: Suggestion : Can you make  "transforms complex JSON data" -> "transforms JSON data" ?

File: docs/usage.md (hunk 1 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md:50]: Fixed

File: README.md (hunk 2 above)
Comment (cs-abhishek-shukla) [reply README.md:19]: It can transform complex JSON as well. ;-)

**Review by cs-suyog-jinturkar — APPROVED** [review]
Yash approved this PR. Approving here as well.

**Pull request merged**